haxe.YAML-tmLanguage

```diff
@@ -24,6 +24,7 @@ repository:
     patterns:
     - include: '#comments'
     - include: '#conditional-compilation'
+    - include: '#method-return-type-hint'
```

Comment:
putting this in `#global` is probably a bit of an overkill, I think `#block-contents` might be a better fit.

Reply:
are there cases where it has to be `#global` or does it not work in `#block-contents`?

Reply:
somehow my vscode checkout still had the old version